media/themes/modern.msstyles/lang/en-GB.rc

```diff
@@ -0,0 +1,39 @@
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_UK
+
+STRINGTABLE
+{
+1000, "Light"
+1001, "Dark"
+}
+
+STRINGTABLE
+{
+2000, "Modern Light"
+2001, "Modern Dark"
+}
+
+STRINGTABLE
+{
+3000, "Regular Fonts"
+3001, "Large Fonts"
+3002, "Extra Large Fonts"
+}
+
+STRINGTABLE
+{
+4000, "Regular Fonts"
+4001, "Large Fonts"
+4002, "Extra Large Fonts"
+}
+
+STRINGTABLE
+{
+5000, "Modern"
+5001, "Visual Style"
+5002, "ReactOS Foundation"
+5003, "Polar, based on win10"
```

Comment:
So is this theme called Polar or Modern after all? Also, saying it's "based on win10" may be misunderstood - maybe this should be worded better, for example "inspired by win10"

Reply:
i was told this was the author string in a previous comment